Libraries/Image/RCTImageLoader.m

```diff
@@ -610,31 +611,26 @@ - (RCTImageLoaderCancellationBlock)decodeImageDataWithoutClipping:(NSData *)data
   }
 }
 
-- (RCTImageLoaderCancellationBlock)getImageSize:(NSString *)imageTag
-                                          block:(void(^)(NSError *error, CGSize size))completionBlock
-{
-  return [self loadImageOrDataWithTag:imageTag
-                                 size:CGSizeZero
-                                scale:1
-                           resizeMode:RCTResizeModeStretch
-                        progressBlock:nil
-                      completionBlock:^(NSError *error, id imageOrData) {
-                        CGSize size;
-                        if ([imageOrData isKindOfClass:[NSData class]]) {
-                          NSDictionary *meta = RCTGetImageMetadata(imageOrData);
-                          size = (CGSize){
-                            [meta[(id)kCGImagePropertyPixelWidth] doubleValue],
-                            [meta[(id)kCGImagePropertyPixelHeight] doubleValue],
-                          };
-                        } else {
-                          UIImage *image = imageOrData;
-                          size = (CGSize){
-                            image.size.width * image.scale,
-                            image.size.height * image.scale,
-                          };
-                        }
-                        completionBlock(error, size);
-                      }];
+- (NSDictionary<NSString *, id>*)getMetadataForImage:(id)imageOrData {
```

Comment:
I believe the name change is just the git diff displaying improperly. The method itself was moved few lines below to be exported (as there's no other place in the codebase using it except the ImageViewManager that no longer has it).